PsfShimMonitor/MainWindow.xaml

```diff
@@ -0,0 +1,403 @@
+                        <MenuItem Name="miKernelProcess"  >
+                            <MenuItem.Header>
+                                <CheckBox Name="cbCatKernelProcess" Content="KernelProcess" IsThreeState="False"
+                                          Checked="Cb_Event_Checked_or_Unchecked" Unchecked="Cb_Event_Checked_or_Unchecked" IsChecked="False"/>
+                            </MenuItem.Header>
+                        </MenuItem>
+                        <MenuItem Name="miKernelImageLoad"  >
+                            <MenuItem.Header>
+                                <CheckBox Name="cbCatKernelImageLoad" Content="KernelImageLoad" IsThreeState="False"
+                                          Checked="Cb_Event_Checked_or_Unchecked" Unchecked="Cb_Event_Checked_or_Unchecked" IsChecked="False"/>
+                            </MenuItem.Header>
+                        </MenuItem>
+                        <MenuItem Name="miKernelDisk"  >
+                            <MenuItem.Header>
+                                <CheckBox Name="cbCatKernelDisk" Content="KernelDiske" IsThreeState="False"
+                                          Checked="Cb_Event_Checked_or_Unchecked" Unchecked="Cb_Event_Checked_or_Unchecked" IsChecked="False"/>
+                            </MenuItem.Header>
+                        </MenuItem>
+                        <MenuItem Name="miKernelFile"  >
+                            <MenuItem.Header>
+                                <CheckBox Name="cbCatKernelFile" Content="KernelFile" IsThreeState="False"
+                                          Checked="Cb_Event_Checked_or_Unchecked" Unchecked="Cb_Event_Checked_or_Unchecked" IsChecked="False"/>
+                            </MenuItem.Header>
+                        </MenuItem>
+                        <MenuItem Name="miKernelRegistry"  >
+                            <MenuItem.Header>
+                                <CheckBox Name="cbCatKernelRegistry" Content="KernelRegistry" IsThreeState="False"
+                                          Checked="Cb_Event_Checked_or_Unchecked" Unchecked="Cb_Event_Checked_or_Unchecked" IsChecked="False"/>
+                            </MenuItem.Header>
+                        </MenuItem>
+
+                        <HeaderedContentControl Header="Other EventType Filters"  Margin="-25,0,0,0" />
+                        <Separator/>
+                        <MenuItem Name="miOther" >
+                            <MenuItem.Header>
+                                <CheckBox Name="cbCatOther" Content="Other"  IsThreeState="False"
+                                          Checked="Cb_Event_Checked_or_Unchecked" Unchecked="Cb_Event_Checked_or_Unchecked" IsChecked="True"/>
+                            </MenuItem.Header>
+                        </MenuItem>
+                    </ContextMenu>
+                </Button.ContextMenu>
+            </Button>
+            <!--            <Grid Grid.Column="4">
```

Comment:
can this be removed?

Reply:
Addressed in commit 727a9e9856d361f686523f2ea2907876c3ef9b23 on 11/28